docs/rgb.md

```diff
@@ -0,0 +1,105 @@
+# RGB/Underglow/Neopixel
+Want your keyboard to shine? Add some lights!
+This does require the neopixel library from Adafruit. This can be downloaded here https://github.com/adafruit/Adafruit_CircuitPython_NeoPixel/blob/master/neopixel.py
+
+Simply put this in the "root" of your circuitpython device. If unsure, it's the folder with main.py in it, and should be the first folder you see when you open the device.
+
+Currently we support the following addressable LEDs:
+
+ * WS2811, WS2812, WS2812B, WS2812C, etc.
+ * SK6812, SK6812MINI, SK6805
+ * All neopixels
+ 
+## Usage
+At minimum you will need to make sure that these are set in either your keymap ip importing an MCU directly, or it should be included in the predefined boards if they support them.
+
+|Define               |Description                                  |
+|---------------------|---------------------------------------------|
+|`keyboard.pixel_pin` |The pin connected to the data pin of the LEDs|
+|`keyboard.num_pixels`|The number of LEDs connected                 |
+
+Then you should be able to use the keycodes below to change the RGB lighting to your liking.
+
+### Color Selection
+
+KMK uses [Hue, Saturation, and Value](https://en.wikipedia.org/wiki/HSL_and_HSV) to select colors rather than RGB. The color wheel below demonstrates how this works.
+
+<img src="gitbook/images/color-wheel.svg" alt="HSV Color Wheel" width="250"/>
```

Comment:
does this link actually go anywhere?.... I don't so far see that file anywhere in the tree

Reply:
No, it does not. Seems like I should add an image here.

Reply:
if you want to be super clever, since the source is an svg, you can just [inline the whole image here](https://www.viget.com/articles/getting-started-with-inline-svg/)